apps/frontend/src/components/user/UpdateUserInformation.tsx

```diff
@@ -350,24 +350,24 @@ export default function UpdateUserInformation(
                 disabled={true}
               />
               <FormikUIAutocomplete
-                name="organisation"
+                name="institutionId"
                 label="Organization"
                 items={institutionsList}
-                data-cy="organisation"
+                data-cy="institution"
                 loading={loadingInstitutions}
                 noOptionsText="No organizations"
               />
-              {+values.organisation === 1 && (
+              {+values.institutionId === 1 && (
                 <>
                   <Field
-                    name="otherOrganisation"
+                    name="otherInstitution"
                     label="Please specify organization"
-                    id="organisation-input"
+                    id="institution-input"
                     type="text"
                     component={TextField}
                     margin="normal"
                     fullWidth
-                    data-cy="otherOrganisation"
+                    data-cy="otherInstitution"
                     required
                   />
                   <FormikUIAutocomplete

```

Comment:
There's a few instances of "organisation" still being used for frontend stuff. Is this intentional?

Reply:
Done